app/plugins/router.js

```diff
@@ -1,14 +1,14 @@
 import health from '../routes/health.js'
 
-const plugin = {
+const router = {
   plugin: {
     name: 'router',
-    register: (server, options) => {
+    register: (server, _options) => {
```

Comment:
Since `options` isn't used, this is changed slightly to `_options` to make the intention explicit as suggested by SonarCloud.